modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -194,6 +194,180 @@ This makes the application more difficult to understand and harder to locate the
+
+The division into modules looks like this:
+
+[source,plaintext]
+----
+application A1
+├── bootstrap (module)
+└── component C1 (module)
+    ├── core (module)
+    │   ├── domain
+    │   ├── port
+    │   │   ├── in
+    │   │   └── out
+    │   ├── usecase
+    │   └── service (optional)
+    └── adapter (module)
+        ├── in
+        │   └── web
+        │       └── (analog with packages)
+        └── out
+            └── repository
+                └── (analog with packages)
+----
+
+Within the application we structure the individual business components using modules.
+The main purpose of this is to have a structure that is future-proof and allows multiple business components to be integrated.
+Modules at this level have the advantage that they can be developed independently and communication must be explicitly regulated.
+For the sake of simplicity, we initially only assume one business component.
+The business component is in turn divided into two further modules `core` and `adapter`.
+This separation allows the dependencies for both modules to be independent of each other and the core to remain free of technical frameworks.
+The direction of communication is also clearly regulated by the dependencies between core and adapter.
+In addition to the module for the business component, there is a `bootstrap` module that can be seen as a starting point for the application.
+When using Spring Boot, the `SpringBootApplication` and configurations can also be found here.
+This module in turn has a dependency on all business components (to be precise on their core and adapter).
+
+With regard to the adapters, the recommendation is to create them within one module in the package structure mentioned above.
+This offers a good compromise between structuring and tailor-made dependencies and manageable complexity on the other hand.
+Given the usual level of complexity, it can be assumed that there are usually no more than 2-3 adapters, so structuring using packages is sufficient here.
+As complexity increases, two additional options become available:
+
+1. Separate modules for in and out. This allows the frameworks for incoming and outgoing adapters to be maintained separately and provides an additional level of structuring.
+2. Individual modules for each adapter. This means that each adapter can be viewed and developed completely isolated.
+The dependencies can also be maintained independently.
+
+With both options, the complexity increases accordingly.
+The recommendation is to start with a package structure and divide it further if necessary.
+
+The management of dependencies should aim to define them as far "down" as possible and not on the root or business component level.
+In particular, technical frameworks such as JPA should be defined directly in the adapter module so that they are not available in the core module.
+In order to keep versions consistent, dependency management in the master pom can be used in Maven.
```

Comment:
Hier wäre ein Link zu Dependency Management in Maven hilfreich für alle, die nicht direkt wissen was gemeint ist.